src/main/java/com/laytonsmith/abstraction/bukkit/entities/BukkitMCLivingEntity.java

```diff
@@ -216,31 +216,32 @@ public void addEffect(int potionID, int strength, int seconds, boolean ambient,
 				le.addPotionEffect(pe, true);
 			}
 		} catch(NullPointerException e){
-			//
 			Logger.getLogger(BukkitMCLivingEntity.class.getName()).log(Level.SEVERE,
 					"Bukkit appears to have derped. This is a problem with Bukkit, not CommandHelper. The effect should have still been applied.", e);
 		}
-//        EntityPlayer ep = ((CraftPlayer) p).getHandle();
-//        MobEffect me = new MobEffect(potionID, seconds * 20, strength);
-//        //ep.addEffect(me);
-//        //ep.b(me);
-//
-//        Class epc = EntityLiving.class;
-//        try {
-//            Method meth = epc.getDeclaredMethod("b", net.minecraft.server.MobEffect.class);
-//            //ep.d(new MobEffect(effect, seconds * 20, strength));
-//            //Call it reflectively, because it's deobfuscated in newer versions of CB
-//            meth.invoke(ep, me);
-//        } catch (Exception e) {
-//            try {
-//                //Look for the addEffect version
-//                Method meth = epc.getDeclaredMethod("addEffect", MobEffect.class);
-//                //ep.addEffect(me);
-//                meth.invoke(ep, me);
-//            } catch (Exception ex) {
-//                Logger.getLogger(BukkitMCPlayer.class.getName()).log(Level.SEVERE, null, ex);
-//            }
-//        }
+	}
+
+	/**
+	 * This calls the 5-param constructor for PotionEffect to disable particles entirely
+	 * @param potionID - ID of the potion
+	 * @param strength - potion strength
+	 * @param seconds - duration of the potion in seconds
+	 * @param ambient - make particles less noticable
+	 * @param particles - enable or disable particles entirely
+	 * @param t - target
+	 */
+	@Override
+	public void addEffect(int potionID, int strength, int seconds, boolean ambient, boolean particles, Target t) {
+		PotionEffect pe = new PotionEffect(PotionEffectType.getById(potionID), (int)Static.msToTicks(seconds * 1000),
+				strength, ambient, particles);
```

Comment:
To support versions prior to 1.8, change this to:
```
PotionEffect pe;
if (Static.getServer().getMinecraftVersion().lt(MCVersion.MC1_8)) {
	pe = new PotionEffect(PotionEffectType.getById(potionID), (int)Static.msToTicks(seconds * 1000),
			strength, ambient);
} else {
	pe = new PotionEffect(PotionEffectType.getById(potionID), (int)Static.msToTicks(seconds * 1000),
			strength, ambient, particles);
}
```
Then there doesn't need to be two different methods, and it won't error on 1.7 if someone tries to use the particles option. There needs to be a similar change on the getEffects method, because a 1.7 server will not like pe.hasParticles(). If the version is less than 1.8, just construct the MCEffect with "true" instead.